content/brazilian-portuguese/jsappybird/index_1.md

```diff
@@ -0,0 +1,264 @@
+---
+title: "Javascript: JSappy Bird"
+description: "Aprenda sobre variáveis, funções e condicionais enquanto construímos um jogo parecido com o Flappy Bird."
+date: 2023-09-11T13:24:17-07:00
+difficulty: "Iniciante/Intermediário"
+draft: true
+hidden: true
+---
+
+![alt text](resources/_gen/images/flappy.png "JSappyBird")
```

Comment:
Can you move all the stuff in the "" into the [] to replace alt text for all cases?

```suggestion
![JSappyBird](resources/_gen/images/flappy.png)
```